### DISCUSSION TITLE
Improvements to class regex

### DISCUSSION THREAD

**Discussion by sannajammeh** [sannajammeh/tw-classed#42] [Ideas]
I'm no regex expert and hope anyone can assist in looking over the classRegex for tailwindcss extension [here](https://tw-classed.vercel.app/docs/vscode-setup)

It is currently working for me, however I'm unsure if its the best way to handle matching.
It should match both `classed(...args)` and `classed.button(...args)`

**Comment by rcoopr** [comment]
The example provided didn't work for the proxy version for me. I used the following

```
["classed(?:\\.\\w*)?\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"]
```

This optionally matches a following `.\w*` after `classed` without capturing it

**Reply by sannajammeh** [reply]
Your regex works excellently! Thank you so much for this. Updating docs.
